homeassistant/components/home_connect/api.py

```diff
@@ -434,26 +456,26 @@ class FridgeFreezer(DeviceWithDoor):
 
     def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
-        door_entity = self.get_door_entity()
-        return {"binary_sensor": [door_entity]}
+        door_entities = self.get_door_entities()
+        return {"binary_sensor": door_entities}
 
 
 class Refrigerator(DeviceWithDoor):
     """Refrigerator class."""
 
     def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
-        door_entity = self.get_door_entity()
-        return {"binary_sensor": [door_entity]}
+        door_entities = self.get_door_entities()
+        return {"binary_sensor": door_entities}
 
 
 class Freezer(DeviceWithDoor):
     """Freezer class."""
 
     def get_entity_info(self):
         """Get a dictionary with infos about the associated entities."""
-        door_entity = self.get_door_entity()
-        return {"binary_sensor": [door_entity]}
+        door_entities = self.get_door_entities()
```

Comment:
more of the same optimization comment eliminating `door_entities`
`return {"binary_sensor": self.get_door_entities()}`